ancestrymanager/ancestrymanager.go

```diff
@@ -208,7 +209,7 @@ func (m *manager) getAncestorsWithCache(key string) ([]string, error) {
 		}
 		project, err := m.resourceManager.Projects.Get(cur).Do()
 		if err != nil {
-			return nil, err
+			return nil, errors.Wrap(err, "for more info- https://cloud.google.com/docs/terraform/policy-validation/troubleshooting#ProjectCallerForbidden")
```

Comment:
A couple things:

1. This wrapping should only happen if this is a 403 error from the API. You will need to use a function like [isGoogleApiErrorWithCode](https://github.com/GoogleCloudPlatform/terraform-validator/blob/07368d5756cbbb7c522f512a5756340a9d14f5b7/converters/google/resources/utils.go#L107) to check that the error is a Google API error and that it has a 403 error code before adding the extra message. (You'll have to copy the logic of that function into the ancestrymanager package, because it's not exported, but that should be fine.) So: a 403 should add this message, but a 500 or a non-API error should not.
2. Instead of using errors.Wrap, could you use `fmt.Errorf("%w: message", err)`? That's the newer way to handle this and makes it a little easier to understand what the final error message will look like.

Reply:
Fixed, Comments incorporated. However in terms of sending the error message, right now I have appended the error message and the helper URL.

`fmt.Errorf("for more info- %v, %v", helperURL, err)`

Instead of sending the original error message should we just send the helper URL. Something like this 

`fmt.Errorf("for more info- %v", helperURL)`

That will make the error output a bit neat but less informational

Reply:
I would be fine with not wrapping the error, but it might be good to make it a little more informational - for example:

```
fmt.Errorf("Project %s does not exist or user does not have the correct permissions. For more info: %s", projectId, helperURL)
```

Reply:
Changed the error message as per suggestion.